src/shared/lib/pathwayMapper/PathwayMapperMessageBox.tsx

```diff
@@ -12,6 +12,7 @@ const PathwayMapperMessageBox: React.FunctionComponent<PathwayMapperMessageBoxPr
     const isWarningMessage = message !== props.loadingMessage;
     return (
         <div
+            id="pathway-mapper-message-box"
```

Comment:
We should use `data-test` prop here:

```suggestion
            data-test="pathwayMapperMessageBox"
```